static/usr/lib/systemd/system/secureboot-db.service.d/core-override.conf

```diff
@@ -1,2 +1,3 @@
-[Unit]
-ConditionKernelCommandLine=snapd_recovery_mode=install
+[Service]
+ExecStart=
+ExecStart=sh -c 'if [ "$(/usr/libexec/core/get-mode mode)" = install ]; then /usr/bin/sbkeysync --no-default-keystores --keystore /usr/share/secureboot/updates --verbose; fi'
```

Comment:
Why did not you use `ExecCondition` here like other places?

Reply:
Right, that is a better option. Changed now, thanks.